service/src/main/kotlin/app/cash/backfila/service/runner/BackfillRunner.kt

```diff
@@ -86,14 +95,14 @@ class BackfillRunner private constructor(
       val batchPrecomputer = BatchPrecomputer(this)
       val batchQueuer = BatchQueuer(this, metadata.numThreads)
       val batchRunner = BatchRunner(
-          this,
```

Comment:
appears y'all don't like continuation indent.. so I applied that style in my intellij. The new ktlint seems to do it more aggresively